core/build.sbt

```diff
@@ -5,6 +5,7 @@ name := "common-core"
 libraryDependencies ++= Seq(
   sprayJson,
   typesafeConfig,
+  "biz.paluch.redis" % "lettuce" % "3.2.Final",
```

Comment:
Why is it that we're switching away from the library @sbhaktha had chosen for the cache?